src/Components/Scenes/Chrome/Search/Search.js

```diff
@@ -14,14 +16,26 @@ import CloseIcon from '@material-ui/icons/Close';
 
 const Search = props => {
   const {
-    searchInput,
     isSearchClicked,
     searchHandleInput,
     handleKeyPress,
     handleSearchClose
   } = props;
 
+  const searchInput = useSelector(
```

Comment:
NitPic: Is there a reason you make it 3 diff selector. Can we make one line selector.
 `const {value,  isError,  isTouched,} = useSelector((state:any) => state.searchInput);`

Reply:
sure i can change that